dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/GeneratorPVAlphaSum.mo

```diff
@@ -0,0 +1,73 @@
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model GeneratorPVAlphaSum "Model for generator PV with AlphaSum."
+
+  extends BaseClasses.BaseGeneratorSimplified;
+  extends AdditionalIcons.Machine;
+
+  public
+
+    parameter Types.PerUnit KGover = 1 "Mechanical power sensitivity to frequency";
+    parameter Types.PerUnit Lambda = 0 "Parameter Lambda of the voltage regulation";
+    parameter Types.ActivePowerPu PMinPu "Minimum active power in p.u (base SnRef)";
+    parameter Types.ActivePowerPu PMaxPu "Maximum active power in p.u (base SnRef)";
+    parameter Types.ReactivePowerPu QMinPu  "Minimum reactive power in p.u (base SnRef)";
+    parameter Types.ReactivePowerPu QMaxPu  "Maximum reactive power in p.u (base SnRef)";
+    parameter Types.ActivePowerPu PAuxHVPu = 0 "Active power consumption of the high voltage auxiliary used to recalculate the P and Q limits when the auxiliaries are not explicitly modeled (base SnRef)";
+    parameter Types.ActivePowerPu QAuxHVPu = 0 "Reactive power consumption of the high voltage auxiliary used to recalculate the P and Q limits when the auxiliaries are not explicitly modeled (base SnRef)";
+    parameter Types.ActivePowerPu PAuxLVPu = 0 "Active power consumption of the low voltage auxiliary used to recalculate the P and Q limits when the auxiliaries are not explicitly modeled  (base SnRef)";
+    parameter Types.ActivePowerPu QAuxLVPu = 0 "Reactive power consumption of the low voltage auxiliary used to recalculate the P and Q limits when the auxiliaries are not explicitly modeled  (base SnRef)";
+    final parameter Types.ActivePowerPu PMinAuxPu = PMinPu - PAuxHVPu - PAuxLVPu "Minimum active power in p.u when taking into account the auxiliaries (base SnRef)";
+    final parameter Types.ActivePowerPu PMaxAuxPu = PMaxPu - PAuxHVPu - PAuxLVPu "Maximum active power in p.u when taking into account the auxiliaries (base SnRef)";
+    final parameter Types.ReactivePowerPu QMinAuxPu = QMinPu - QAuxHVPu - QAuxLVPu "Minimum reactive power in p.u when taking into account the auxiliaries (base SnRef)";
+    final parameter Types.ReactivePowerPu QMaxAuxPu = QMaxPu - QAuxHVPu - QAuxLVPu "Maximum reactive power in p.u when taking into account the auxiliaries (base SnRef)";
+
+    parameter Real AlphaRaw "Reference value of the participation of the considered generator in the frequency regulation";
+
+    Connectors.ImPin N "Signal to change the active power reference setpoint of all the generators in the system in p.u (base SnRef)";
+    Connectors.ZPin Alpha "Participation of the considered generator in the frequency regulation";
```

Comment:
maybe add a comment? equals to AlphaRaw if the group is not limited, 0 otherwise?

Reply:
You are right. Done.